src/service.js

```diff
@@ -39,11 +39,11 @@ class Service {
 	 * Creates an instance of Service by schema.
 	 *
 	 * @param {ServiceBroker} 	broker	broker of service
-	 * @param {Object} 			schema	schema of service
+	 * @param {Object=} 		schema	schema of service
 	 *
 	 * @memberof Service
 	 */
-	constructor(broker, schema) {
+	constructor(broker, schema = undefined) {
```

Comment:
Why need it `schema = undefined`. I think it is not relevant, because if you don't define˙it, it will be `undefined`

Reply:
My IDE highlit this. I just added this for comfortable IDE workaround. If it not required, I can remove this

Reply:
OK.

Reply:
@icebob I reverted changes in `service.js` file. Right now only related to feature changes